app/controllers/v2/event_invitations_controller.rb

```diff
@@ -21,11 +21,32 @@ def create
 
     def send_notifications(event_invitation)
       event_invitation.email_addresses_to_array.each do |email_address|
-        EventInvitationMailer.invite(
-          email_address: email_address,
-          event: event_invitation.event,
-          person: Person.find_by(email_address: email_address)
-        ).deliver_later
+        person = Person.find_by(email_address: email_address)
+
+        if person
+          send_email_notification(event_invitation, person)
+          send_sms_notification person
+        end
```

Comment:
hmm, was assuming we'd always send the email notification, and an sms as well if opted. @cromulus @asyp would you please confirm what it should be?